test/ProgressOnderwijsUtils.Tests/HtmlTests.cs

```diff
@@ -121,5 +121,15 @@ public void AllInterestingPlusOperatorOrderingsCompileAndReturnHtmlFragment()
             var typeReflectionHack = new { a, b, c, d, e, f, g, h, i, j, k, l, m, n, o };
             PAssert.That(() => typeReflectionHack.GetType().GetProperties().Select(fi => fi.PropertyType).Distinct().SequenceEqual(new[] { typeof(HtmlFragment) }));
         }
+
+        [Fact]
+        public void SetOfEmptyHtmlFragmentsIsEmpty()
+        {
+            var fragment = HtmlFragment.Fragment(HtmlFragment.Empty, HtmlFragment.Empty);
+            PAssert.That(() => fragment.IsEmpty);
+
+            var fragmentWithArray = HtmlFragment.Fragment(new[]{ HtmlFragment.Empty, HtmlFragment.Empty });

```

Comment:
ivm params array is dit exact hetzelfe; deze tweede assert lijkt me niet zinnig.

Echter lijkt het me wel zinnig om een assert toe te voegen voor Array.Empty, en voor een mengelmoes uit HtmlFragment.Empty en "", en evt. nog iets met JoinHtml().

Reply:
> ivm params array is dit exact hetzelfe; deze tweede assert lijkt me niet zinnig.

Echter ging de eeste assert goed en de 2e fout tot ik de fix met `.All(element => element.IsEmpty)` deed.

Reply:
Er was een overload dus ik heb ze samengevoegd.

Reply:
Ah, de overload maakt het dus wel nuttig!